dolphinscheduler-task-plugin/dolphinscheduler-task-seatunnel/src/main/java/org/apache/dolphinscheduler/plugin/task/seatunnel/SeatunnelTask.java

```diff
@@ -154,26 +155,21 @@ private String buildCommand() throws Exception {
 
     protected List<String> buildOptions() throws Exception {
         List<String> args = new ArrayList<>();
+        args.add(CONFIG_OPTIONS);
```

Comment:
Please add some UT for this.

Reply:
Done, Update the SeatunnelTaskTest, PTAL, thanks